Practice-Problems/zip-code-validator/main.js

```diff
@@ -0,0 +1,12 @@
+function zipCodeValidator(zipToValidate){
+  if (zipToValidate.length === 5){
+    !isNaN(zipToValidate);
```

Comment:
This line is not in a conditional block, so it has no influence on the line below it. As it is right now, you are returning true as long as the length is 5, but you are never checking that it is a number. Try calling your function with a five letter word, and you should get back true, which is not the intention.

Reply:
My bad, I have updated it as the conditional check and since the value was able to return as truthy or falsy value, I removed if statement and direct return the value. new commit c0c602a